cub/test/catch2_test_device_transform.cu

```diff
@@ -388,6 +389,12 @@ C2H_TEST("DeviceTransform::Transform add five streams", "[device][device_transfo
   c2h::device_vector<std::int64_t> d(num_items, 4);
   c2h::device_vector<float> e(num_items, 5);
 
+  c2h::gen(C2H_SEED(1), a, std::int8_t{10}, std::int8_t{100});
+  c2h::gen(C2H_SEED(1), b, std::int16_t{10}, std::int16_t{100});
+  c2h::gen(C2H_SEED(1), c, std::int32_t{10}, std::int32_t{100});
+  c2h::gen(C2H_SEED(1), d, std::int64_t{10}, std::int64_t{100});
+  c2h::gen(C2H_SEED(1), e, float{10}, float{100});
```

Comment:
suggestion: this change overwrites values initially stored in these vectors. I'd suggest to remove initial value from vector constructors to save some mental load when reading this test (are these values left there for a purpose? etc.)